users/services.py

```diff
@@ -0,0 +1,53 @@
+from django_ilmoitin.utils import send_notification
+
+from users.models import Guardian
+from users.notifications import NotificationType
+from users.utils import get_communication_unsubscribe_ui_url
+
+
+class GuardianEmailManagementNotificationService:
```

Comment:
```suggestion
class GuardianEmailNotificationService:
```
The class name already tells it's about emails and notifications so
IMHO removing `Management` would be beneficial as it'd make the
class name a bit shorter. How do you see this?

Reply:
Ah, so how about `GuardianEmailChangeNotificationService` to make it explicit what's it for or adding some documentation to the class to indicate its purpose?

Reply:
I can change this, but I personally feel that the Management was much more professional